README.md

```diff
@@ -117,4 +109,18 @@ It is **highly encouraged** to use the [command line interface](https://github.c
 - Pull up on [issues](https://github.com/sern-handler/handler/issues) and report bugs
 - All kinds of contributions are welcomed.
 
+## Build
 
+- Install pnpm
+```sh
+npm install -g pnpm
+```
+- Copy
+```sh
+git clone https://github.com/sern-handler/handler.git
+```
+- Create bundle
+```sh
+//prod for production, dev for dev build
+pnpm run build:(prod|dev)
+```
```

Comment:
Do you really think people will primarily focus on these? We can also include these in docs

Reply:
Its for those who want to contribute and quick-start build. Do you think we should move this somewhere else?

Reply:
Yeah it'll fit better, we may lose some users due to a long README section

Reply:
Ok. I was thinking I can link how to build, which will redirect to the website. how does that sound?

Reply:
That works, yeah.